torchaudio/transforms/_transforms.py

```diff
@@ -355,6 +355,13 @@ class MelScale(torch.nn.Module):
             (area normalization). (Default: ``None``)
         mel_scale (str, optional): Scale to use: ``htk`` or ``slaney``. (Default: ``htk``)
 
+    Example
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
+        >>> spectrogram_transform = torchaudio.transforms.Spectrogram(n_fft=1024)
```

Comment:
for code consistency,
```suggestion
        >>> spectrogram_transform = transforms.Spectrogram(n_fft=1024)
```